src/co/elastic/NotificationManager.groovy

```diff
@@ -422,6 +424,18 @@ def generateBuildReport(Map args = [:]) {
     return output
 }
 
+def snapshotyUrl() {
+  def short_commit = env.GIT_BASE_COMMIT.substring(0, Math.min(env.GIT_BASE_COMMIT.length(), 8));
+  def filter = "(%22StorageObjectListTable%22:(%22f%22:%22%255B%257B_22k_22_3A_22Name%2520contains_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_${short_commit}_5C_22_22_2C_22i_22_3A_22displayName_22%257D%255D%22))"
```

Comment:
NIP: I'd use a decoded the URL to make it clear. And then encode it before usage.

Reply:
Good point! I tried to use the encode/decode but somehow something didn't work nicely in my end. So I'd keep as it is since we are deprecating this particular Jenkins library to use GH actions instead